### PR TITLE
Created commands encode and decode

### PR DESCRIPTION
**Pull request by Gladhus**
I created two new commands (encode and decode) so that we can now add multiple encoding and decoding. Merged the Base64 into the encode and decode methods, and added the binary encoding and decoding.

### CODE REVIEW

File: scripts/encode.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	for (var i = 0;i < length; i++) {
+		var bin = msg[i].charCodeAt().toString(2);
+		output.push(Array(8-bin.length+1).join("0") + bin);
+	}
+	msg.send(output.join(" "));
+}
+
+module.exports = function(robot)
+{
+	robot.respond(/encode base34 (.*)/i, utf8_to_base64);
```
Comment (davidlebr1) [inline scripts/encode.js]: base34 change for base64 ;)

Comment (davidlebr1) [inline scripts/encode.js]: Fix base34 to base64 ;)

File: scripts/encode.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	text = msg.match[1];
+	for (var i = 0;i < length; i++) {
+		var bin = msg[i].charCodeAt().toString(2);
+		output.push(Array(8-bin.length+1).join("0") + bin);
+	}
+	msg.send(output.join(" "));
+}
+
+module.exports = function(robot)
+{
```
Comment (davidlebr1) [inline scripts/encode.js]: Fix base34 to base64 ;)

Comment (Gladhus) [inline scripts/encode.js]: done

File: scripts/decode.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+// Description:
+//   Decodes text in different format
+//	 Current encryptions:
```
Comment (pric) [inline scripts/decode.js]: Indentation

File: scripts/encode.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+//
+// Configuration:
+//   None
+//
+// Commands:
+//   hackerman encode <encryption> <string> - Encodes string
+//																							in the encryption chosen.
+//
+// Authors:
+//	 Charles-Antoine Chartrand
```
Comment (pric) [inline scripts/encode.js]: Indentation

**Pull request merged**